xbmc/video/windows/GUIWindowVideoNav.cpp

```diff
@@ -491,14 +492,24 @@ bool CGUIWindowVideoNav::GetDirectory(const std::string &strDirectory, CFileItem
     }
 
     CVideoDbUrl videoUrl;
-    if (videoUrl.FromString(items.GetPath()) && items.GetContent() == "tags" &&
-       !items.Contains("newtag://" + videoUrl.GetType()))
+    if (videoUrl.FromString(items.GetPath()))
     {
-      CFileItemPtr newTag(new CFileItem("newtag://" + videoUrl.GetType(), false));
-      newTag->SetLabel(g_localizeStrings.Get(20462));
-      newTag->SetLabelPreformatted(true);
-      newTag->SetSpecialSort(SortSpecialOnTop);
-      items.Add(newTag);
+      if (items.GetContent() == "tags" && !items.Contains("newtag://" + videoUrl.GetType()))
+      {
+        const auto newTag{std::shared_ptr<CFileItem>(new CFileItem("newtag://" + videoUrl.GetType(), false))};
```

Comment:
Please use `std::make_shared` to create the instance.

=> `const auto newTag{std::make_shared<CFileItem>("newtag://" + videoUrl.GetType(), false)};`